### ISSUE TITLE
Different Result-views

### ISSUE THREAD

**Issue by jotoeri** [nextcloud/forms#314]
@jancborchardt @skjnldsv 
Ok, so to clarify, these were the views, that i had in mind. As i understood, i think we agreed, that we need both UI-views?

The radio-buttons now were not the way to go i heard from you., So i think the main discussion-point now was how to switch between the two main-contents? Google does with the three tabs at the top...?

Sorry, @jancborchardt  for that strange discussion, after talking to John i think we just talked particularly on completely different directions & things... ;)

![grafik](https://user-images.githubusercontent.com/47433654/80615880-8a3cc580-8a40-11ea-9414-fc9a8b20fb09.png)
![grafik](https://user-images.githubusercontent.com/47433654/80615939-9aed3b80-8a40-11ea-921c-77151e7dd015.png)
![grafik](https://user-images.githubusercontent.com/47433654/80616565-5a41f200-8a41-11ea-9952-be55f22f4dc6.png)

**Comment by skjnldsv** [comment]
So, what about some submenus on the navigation to differenciate the three views?
I would only show them when you are in the results page though.
![image](https://user-images.githubusercontent.com/14975046/80619469-06d1a300-8a45-11ea-8ca3-bd72e9b1e152.png)

**Comment by jotoeri** [comment]
Sounds in general good to me. Just wouldn't in consequence then things like 'edit' & 'copy Share Link' be necessary in this menu, too? So these response-uis would become a submenu of 'Responses'? But a subsub-menu would be too much, i think...?? 🤔

**Comment by skjnldsv** [comment]
Not really because they're all part of the Responses view. Default would be "summary" I guess

**Comment by jotoeri** [comment]
Ah, so you show the submenu in navigation only in results-UI?

**Comment by jancborchardt** [comment]
Let’s keep it simple and visible:
- It is not necessary to put the view switch options in the sidebar. We can simply put it below the h2, like the "Export to CSV" button is now. The new primary action would be to switch between "Summary" and "Individual", and "Export" would be another button.
- No submenus in the navigation are needed here, it’s way more complex than needed. We already have the 3-dot menu, and it makes no sense to have a hierarchy there.


TL;DR: @jotoeri your mockups are good, and let’s move all the stuff from the sidebar into the content view below the h2, making the sidebar not needed in the results view.
